.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-<!-- Thank you for creating a pull request! In general, we merge your pull requests after they get two or more approvals. -->
+<!-- Thank you for creating a pull request! In general, we merge your pull request after it gets two or more approvals. To proceed to the review process by the maintainers, please make sure that the PR meets the following conditions: (1) it passes all CI checks, and (2) it is neither in the draft nor WIP state. If you wish to discuss the PR in the draft state or need any other help, please mention the Optuna development team in the PR. -->
```

Comment:
[Question] I guess we don't have the development team in the Optuna org. How do contributors find the names of the maintainers?

> please mention the Optuna development team in the PR

Reply:
Right; so far, the contributors have to guess watching the reviewers of other PRs. As a followup task, we've better create a team or provide an official list of maintainers somewhere in the repository.